python/NEXT.md

```diff
@@ -0,0 +1,41 @@
+# Next
+Some thoughts on the next steps for refactoring.
+
+In general there should an iterative refactoring of legacy updater. In which Item type should be
+refactored one by one.
+
+## Side effects
+The main interface for updating uses side effects. Meaning that the update doesn't return a new list but updates the old list.
+
+In my experience side effects should be avoided as they make code bases harder to understand and test.
+
+It might then be an idea the update be changed so that it returns a new list of Items.
```

Comment:
Nit: phrase isn't properly formed. Suggestion: "It might then be a beneficial idea to change the update functionality so that it returns a..."

Reply:
https://github.com/benjamincerigo/GildedRose-Refactoring-Kata/pull/1/commits/9e83987e14ecfdf66984124043965d4bdabfd8c2